src/hw_cfgs.ini

```diff
@@ -8,6 +8,8 @@ interconnectpitch = 100
 transistorsize = 3
 # cache size in MB
 cachesize = 1
+memsize = 2
```

Comment:
Is this GB?
Let's have memsize not be a param that's set in config instead it's put into hardware model when we call `init_memory`